core/domain/email_manager.py

```diff
@@ -374,39 +374,32 @@ def require_valid_intent(intent):
         raise Exception('Unrecognized email intent: %s' % intent)
 
 
-def _get_email_config(intent):
-    """Return the default body for the email type matching the given moderator
-    action intent.
-
-    Args:
-        intent: str. The intent string (cause/purpose) of the email.
+def _get_email_config():
```

Comment:
Is this just for moderators? If so, maybe just inline it or put "moderator" in its name?